src/locale/messages.sv.xlf

```diff
@@ -435,202 +435,196 @@
-      <trans-unit id="90e55bb71b57ffa3d778c6a9e676c505d4b897cb" datatype="html">
-        <source>Users who just want to know whether the domain name they own or use have any issues or not.</source>
-        <target state="new">Users who just want to know whether the domain name they own or use have any issues or not.</target>
+      <trans-unit id="07641f98e2977d5ceaea98ceac94a82f0787e6fc" datatype="html">
+        <source>users who want to know whether the domain name they own or use have any issues or not.</source>
+        <target state="new">Användare som bara vill veta om domäner som hen äger eller använder har några problem eller inte.</target>
         <note priority="1" from="description">faq answer</note>
       </trans-unit>
       <trans-unit id="7f82cd00cb66740b1e8e6695ceb6987d951e38e5" datatype="html">
         <source>Users of the second category should contact their DNS operator in case there are errors or warnings for any test of their domain name.</source>
-        <target state="new">Users of the second category should contact their DNS operator in case there are errors or warnings for any test of their domain name.</target>
+        <target state="translated">Användare av den andra kategorin bör vända sig till sin DNS-operatör ifall det finns fel eller varningar när deras domännamn testats.</target>
         <note priority="1" from="description">faq answer</note>
       </trans-unit>
       <trans-unit id="8dbcc14f18696f9d909524ca7364f9988cf901b8" datatype="html">
         <source>What makes Zonemaster different from other DNS zone validating software?</source>
-        <target state="new">What makes Zonemaster different from other DNS zone validating software?</target>
+        <target state="translated">Vad skiljer Zonemaster från andra program som testar domäner?</target>
         <note priority="1" from="description">faq question</note>
       </trans-unit>
-      <trans-unit id="3942f9186d69777597c1fba63f78762033da2ee4" datatype="html">
-        <source>Firstly, Zonemaster saves all history from earlier tests based on the tested domain name, which means you can go back to a test you did some time ago and compare it to the test you ran just a moment ago.</source>
-        <target state="new">Firstly, Zonemaster saves all history from earlier tests based on the tested domain name, which means you can go back to a test you did some time ago and compare it to the test you ran just a moment ago.</target>
+      <trans-unit id="c625cfabcffbab7888c7bb7b57387ef5921eaf17" datatype="html">
+        <source>Zonemaster saves all history from earlier tests based on the tested domain name, which means you can go back to a test you did some time ago and compare it to the test you ran just a moment ago.</source>
+        <target state="new">För det första så sparar Zonemaster all testhistorik utifrån den testade domänen, vilket innebär att man kan gå tillbaka till ett tidigare test och jämföra med ett test som just har körts.</target>
```

Comment:
not the same id and missing `Firstly,...` on `<source>`.

Reply:
Both are expected, `Firstly` (and all subsequent secondary etc) has been remove (FAQ text has been updated in #443, see https://github.com/zonemaster/zonemaster-gui/pull/443/files#diff-28962e2cfc5a8125f80d1a904b7262bbc8af8173e16c8197894cd75167b35de3L63), the ID is based on the content so it is normal that it has changed.